recipes/mknotebooks/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "mknotebooks" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/greenape/mknotebooks/archive/{{ version }}.tar.gz
+  sha256: b8bcc546fa60b2d78511d18585d1525ecbbabda689fa7b038386eebc62296d02
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
```

Comment:
since this is not noarch, the skip line should be `skip: true  # [win or py<36]`

Reply:
Thanks